### DISCUSSION TITLE
Missing files on build

### DISCUSSION THREAD

**Discussion by marinusmaurice** [dotnet/winforms#6303] [Q&A]
* .NET Core Version: 
.NET SDK (reflecting any global.json):
 Version:   6.0.100
 Commit:    9e8b04bbff

Runtime Environment:
 OS Name:     debian
 OS Version:  11
 OS Platform: Linux
 RID:         debian.11-x64
 Base Path:   /usr/share/dotnet/sdk/6.0.100/

Host (useful for support):
  Version: 6.0.0
  Commit:  4822e3c3aa

.NET SDKs installed:
  6.0.100 [/usr/share/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.0 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.0 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

* Have you experienced this same bug with .NET Framework?: 
Not related to .Net Framework

**Problem description:**

I forked the dotnet/winforms repo.
Then ran in repo root directory
 ./build.sh -t --verbosity d

The following errors occur:

       "/home/[user]/.nuget/packages/microsoft.dotnet.arcade.sdk/7.0.0-beta.21576.4/tools/Build.proj" (default target) (1) ->                                         
       "/home/[user]/repos/winforms/Winforms.sln" (Build;Test target) (3:2) ->                                                                              
       "/home/[user]/repos/winforms/pkg/Microsoft.Private.Winforms/Microsoft.Private.Winforms.csproj" (default target) (11:6) ->                            
       "/home/[user]/repos/winforms/src/System.Windows.Forms/src/System.Windows.Forms.csproj" (GetPackageContent target) (7:55) ->                          
       (GetPackageContent target) ->                                                                                                                                   
         /home/[user]/repos/winforms/eng/packageContent.targets(105,5): error : System.Windows.Forms.xml was not found in Microsoft.Private.Intellisense package. Contact the intellisense team about adding the docs for this assembly. [/home/[user]/repos/winforms/src/System.Windows.Forms/src/System.Windows.Forms.csproj]                                                                                                                                                                

       "/home/[user]/.nuget/packages/microsoft.dotnet.arcade.sdk/7.0.0-beta.21576.4/tools/Build.proj" (default target) (1) ->                                         
       "/home/[user]/repos/winforms/Winforms.sln" (Build;Test target) (3:2) ->                                                                              
       "/home/[user]/repos/winforms/pkg/Microsoft.Private.Winforms/Microsoft.Private.Winforms.csproj" (default target) (11:6) ->                            
       "/home/[user]/repos/winforms/src/System.Windows.Forms.Primitives/src/System.Windows.Forms.Primitives.csproj" (GetPackageContent target) (30:38) ->
         /home/[user]/repos/winforms/eng/packageContent.targets(105,5): error : System.Windows.Forms.Primitives.xml was not found in Microsoft.Private.Intellisense package. Contact the intellisense team about adding the docs for this assembly. [/home/[user]/repos/winforms/src/System.Windows.Forms.Primitives/src/System.Windows.Forms.Primitives.csproj]

       "/home/[user]/.nuget/packages/microsoft.dotnet.arcade.sdk/7.0.0-beta.21576.4/tools/Build.proj" (default target) (1) ->
       "/home/[user]/repos/winforms/Winforms.sln" (Build;Test target) (3:2) ->
       "/home/[user]/repos/winforms/pkg/Microsoft.Private.Winforms/Microsoft.Private.Winforms.csproj" (default target) (11:6) ->
       "/home/[user]/repos/winforms/src/Microsoft.VisualBasic.Forms/src/Microsoft.VisualBasic.Forms.vbproj" (GetPackageContent target) (54:10) ->
         /home/[user]/repos/winforms/eng/packageContent.targets(105,5): error : Microsoft.VisualBasic.Forms.xml was not found in Microsoft.Private.Intellisense package. Contact the intellisense team about adding the docs for this assembly. [/home/[user]/repos/winforms/src/Microsoft.VisualBasic.Forms/src/Microsoft.VisualBasic.Forms.vbproj]

       "/home/[user]/.nuget/packages/microsoft.dotnet.arcade.sdk/7.0.0-beta.21576.4/tools/Build.proj" (default target) (1) ->
       "/home/[user]/repos/winforms/Winforms.sln" (Build;Test target) (3:2) ->
       "/home/[user]/repos/winforms/pkg/Microsoft.Private.Winforms/Microsoft.Private.Winforms.csproj" (default target) (11:6) ->
       "/home/[user]/repos/winforms/src/System.Windows.Forms.Design/src/System.Windows.Forms.Design.csproj" (GetPackageContent target) (10:24) ->
         /home/[user]/repos/winforms/eng/packageContent.targets(105,5): error : System.Windows.Forms.Design.xml was not found in Microsoft.Private.Intellisense package. Contact the intellisense team about adding the docs for this assembly. [/home/[user]/repos/winforms/src/System.Windows.Forms.Design/src/System.Windows.Forms.Design.csproj]

       "/home/[user]/.nuget/packages/microsoft.dotnet.arcade.sdk/7.0.0-beta.21576.4/tools/Build.proj" (default target) (1) ->
       "/home/[user]/repos/winforms/Winforms.sln" (Build;Test target) (3:2) ->
       "/home/[user]/repos/winforms/pkg/Microsoft.Private.Winforms/Microsoft.Private.Winforms.csproj" (default target) (11:6) ->
       "/home/[user]/repos/winforms/src/Accessibility/src/Accessibility.ilproj" (GetPackageContent target) (53:32) ->
         /home/[user]/repos/winforms/eng/packageContent.targets(105,5): error : Accessibility.xml was not found in Microsoft.Private.Intellisense package. Contact the intellisense team about adding the docs for this assembly. [/home/[user]/repos/winforms/src/Accessibility/src/Accessibility.ilproj]
    0 Warning(s)
    5 Error(s)

Time Elapsed 00:00:43.86
Build failed with exit code 1. Check errors above.

**Comment by merriemcgaw** [comment]
@RussKie can you take a look when you get online?

**Comment by RussKie** [comment]
**The official disclaimer:** Windows Forms (as the name implies) is a Windows only technology, and we do not provide support for building on other platforms.

With that out of the way, can you provide the full output log? 
When a build is run it outputs all xml-doc files into `$(ArtifactsBinDir)/docs`, and if those files are absent you get the error you're observing.
https://github.com/dotnet/winforms/blob/9cc28d1813505c1369f574ad01a4b2b5036acb74/eng/packageContent.targets#L6
https://github.com/dotnet/winforms/blob/9cc28d1813505c1369f574ad01a4b2b5036acb74/eng/packageContent.targets#L91
https://github.com/dotnet/winforms/blob/9cc28d1813505c1369f574ad01a4b2b5036acb74/eng/packageContent.targets#L100-L107
